analytics/js_tracking_api.rst

```diff
@@ -536,36 +535,36 @@ Use it as a function inside listener on event::
 
 .. describe:: contentTarget
 
-    **Required** ``string`` Url of Content Impression Target
+    **Required** ``string`` URL of Content Impression Target
 
 Example of use::
 
     _paq.push(["trackContentImpression", "clicked", "trackingWhitepaper", "document", "http://cooltracker.tr/whitepaper"]);
 
 .. warning::
-    Use this function in conjunction with ``trackContentImpression`` as it can only be mapped with an impression by linking ``contentName``
-    it's not mapping automatically as  ``trackContentInteractionNode``.
+    Use this function in conjunction with ``trackContentImpression``, as it can only be mapped with an impression by linking ``contentName``.
+    It does not map automatically as  ``trackContentInteractionNode``.
```

Comment:
@wronan any comment?

Reply:
```trackContentInteraction``` and ```trackContentImpression``` do what they say (what they are called: impression and interaction) manually. It says 'in conjunction' because one would probably first track impression of some element, then interaction with it (if any). This relations is built upon given contentName parameter value. And yes, this doesn't cause listeners to be set.